### DISCUSSION TITLE
How to use dioxus+axum+sqlx+mysql to build an information management system, is there a tutorial?

### DISCUSSION THREAD

**Discussion by zuijiaoghb** [DioxusLabs/dioxus#2742] [Q&A]
it is already: I use it every day.
Maybe you meant something else than [this](https://dioxuslabs.com/learn/0.5/reference/fullstack)?

_Originally posted by @ThomasCartier in https://github.com/DioxusLabs/dioxus/issues/2727#issuecomment-2256307691_

**Comment by chungwong** [comment]
I believe this is not an issue but a question should be asked either in https://github.com/DioxusLabs/dioxus/discussions or on the Discord server

**Comment by ealmloff** [comment]
I haven't seen a tutorial that exactly fits that description, but we do have an example that integrates dioxus fullstack, axum and sqlx with authentication [here](https://github.com/DioxusLabs/dioxus/tree/main/packages/fullstack/examples/axum-auth)
